src/main/webapp/js/courseSelector.js

```diff
@@ -15,39 +15,76 @@ export const CollegePlanner = (() => {
   let selected = [];  // Courses selected by the user
   let courses = [];   // List with all courses
   /**
-   * @type {{courseId: string}, {courseName: string}}
+   * @type {{courseId: string}, {course: Object}}
    */
-  const courseNames = {};
+  const courseInfo = {};
 
   /**
-   * Gets courses from /courselist servlet to populate dropdown list
+   * Gets departments from /api/departments servlet to populate dropdown list
    */
-  async function getOptions() {
-    const response = await fetch('/api/courses');
+  async function getDepartmentOptions() {
+    const response = await fetch('/api/departments');
```

Comment:
a note for the future - we'd likely want to `try`/`catch` the fetches, and do proper error handling here too. We can file this as a future improvement.